JDI.Light/JDI.Light/Elements/Complex/Table/Line.cs

```diff
@@ -1,15 +1,15 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
-using JDI.Light.Elements.Base;
+using OpenQA.Selenium;
 
 namespace JDI.Light.Elements.Complex.Table
 {
     public class Line
     {
         private Dictionary<string, string> _data;
-        private readonly List<UIElement> _elements;
-        private readonly List<string> _headers;
+        private static IEnumerable<IWebElement> _elements;
+        private static List<string> _headers;
```

Comment:
please do not create a static members inside a class that can have several instances